rusoto/services/events/README.md

```diff
@@ -1,6 +1,6 @@
 
 # Rusoto CloudWatchEvents
-Rust SDK for Amazon CloudWatch Events
+Rust SDK for Amazon EventBridge
```

Comment:
@matthewkmayer do you think, in this special case the [client name](https://github.com/rusoto/rusoto/blob/master/service_crategen/services.json#L372) should change to EventBridge? The name changed [here](https://github.com/boto/botocore/commit/d968e9ab77880ee186be1078f44f9e9b08b3bbc5) in botocore.

Reply:
Let's do the rename to keep our behavior in line with official SDKs. 👍

Reply:
Will do. I'll do one last sync with boto core and ping after